dockerfiles/vcpkg/ubuntu-22.10.dockerfile

```diff
@@ -1,20 +1,6 @@
 ARG VCPKG_REF=latest
 FROM hpccsystems/platform-build-base-ubuntu-22.10:$VCPKG_REF
 
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    default-jdk \
-    ninja-build \
-    python3-dev \
-    rsync \
-    fop \
-    libsaxonb-java \
-    r-base \
-    r-cran-rcpp \
-    r-cran-rinside \
-    r-cran-inline
-
-RUN git config --global --add safe.directory '*'
-
-WORKDIR /hpcc-dev
-
 ENTRYPOINT ["/bin/bash", "--login", "-c"]
+
```

Comment:
I'm assuming we moved these additions to the dockerfile image over into the vcpkg repository dockerfile/ files?

Reply:
Yes - its only a convenience for testing, both 9.4.x and 9.6.x are setup this way already.